src/System.Private.CoreLib/shared/Internal/Win32/RegistryKey.cs

```diff
@@ -219,7 +220,8 @@ public unsafe string[] GetValueNames()
             return GetValue(name, null);
         }
 
-        public object? GetValue(string name, object? defaultValue) // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/26761
+        [return: NotNullIfNotNull("defaultValue")]
```

Comment:
can these use nameof?

Reply:
> can these use nameof?

Unfortunately, no.  The parameter names are not in scope here.